### PR TITLE
2309 Completed Order Should Break Out Items by Shops

### LINKED ISSUE #2309: Completed order template should break items out by shipment
The order completed template that is shown to customer when an order has been completed or when viewing historical orders should split the display of line items, shipment by shipment basis for orders with multiple shipments.

If the order is from a multi-vendor shop, and there is only one shipment per vendor, the vendor breakout and the shipment breakout can be noted together.

The shipment breakout should identify the expected arrival date of the items along with any tracking information if the order has been shipped.

If the order has not been shipped, it should identify any issues fulfilling the order, such as items on backorder and delays or closures in the warehouse along with an expected shipping date.

### PR DESCRIPTION
**Pull request by abdulsemiu-atanda**
Resolves #2309 

**Testing**
- Clone this Branch and run Reaction with `reaction reset -n && reaction`
- Login with `admin` credentials
- Register and Login as a user in another browser
- Go to the user's profile and select `Become A Seller`
- Go to the browser where the `admin` is logged in and grant the new user permission to create product
- Create product with the user and publish
- Using any Logged on User (i.e `Admin` or `Seller`) of your choice and add at least one product from the new seller shop and default shop (`Reaction`) to your `Cart`
- Checkout items in your `Cart`
- Observe the new design on the `OrderCompleted` page and how items are displayed by grouping them by shops

### CODE REVIEW

**Comment by abdulsemiu-atanda** [comment]
Please note that `shop/createShop` method is still faulty and as such this can't be tested with multiple shops yet but I've opened issue #2495. However, the new design can be tested by checking out products from the existing default shop

**Comment by abdulsemiu-atanda** [comment]
@rymorgan you can check out the design implementation of this PR. Thanks

**Comment by rymorgan** [comment]
@abdulsemiu-atanda here are my design notes. 
cart_completed-1
cart_completed-responsive

**Comment by rymorgan** [comment]
Have some more notes on this, we still aren't quite
cart_completed-notes
 there yet. 

Also the mobile version still isn't working right. It should look like this:
https://zpl.io/Z1qVSB9

**Comment by abdulsemiu-atanda** [comment]
@rymorgan great. I'll make sure to fix what's left

File: imports/plugins/core/orders/client/components/orderListItems.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { Component, PropTypes } from "react";
```
Comment (zenweasel) [inline imports/plugins/core/orders/client/components/orderListItems.js]: same comment here about `PropTypes`

File: imports/plugins/core/orders/client/containers/orderListItemsContainer.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    this.state = {
+      shopItemMap: {}
+    };
+  }
+  componentWillMount() {
+    this.setState({ shopItemMap: this.handleMapShopsToItems() });
+    shopsToItemsMap = {};
+  }
+  handleImage(item) {
+    const itemImageSub = Meteor.subscribe("CartItemImage", item);
```
Comment (zenweasel) [inline imports/plugins/core/orders/client/containers/orderListItemsContainer.js]: Missing import

**Comment by zenweasel** [comment]
@abdulsemiu-atanda What's the status on this? Should I have someone else pick this up and finish it?

**Comment by zenweasel** [comment]
Closing in favor of #2532

**Pull request closed without merging**